printer_zpl2/tests/test_wizard_import_zpl2.py

```diff
@@ -0,0 +1,94 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestWizardImportZpl2(TransactionCase):
+    def setUp(self):
+        super(TestWizardImportZpl2, self).setUp()
+        self.Model = self.env['wizard.print.record.label']
+        self.server = self.env['printing.server'].create({})
+        self.printer = self.env['printing.printer'].create({
+            'name': 'Printer',
+            'server_id': self.server.id,
+            'system_name': 'Sys Name',
+            'default': True,
+            'status': 'unknown',
+            'status_message': 'Msg',
+            'model': 'res.users',
+            'location': 'Location',
+            'uri': 'URI',
+        })
+        self.label = self.env['printing.label.zpl2'].create({
+            'name': 'ZPL II Label',
+            'model_id': self.env.ref(
+                'base_report_to_printer.model_printing_printer').id,
+        })
+
+    def test_open_wizard(self):
+        """ open wizard from label"""
+        self.label.import_zpl2()
```

Comment:
Please add a simple assert to check the returned dictionnary (or at least some values like the model).